test/integration/stateView/document/mongoDbRepository/SVDocumentMongoDbRepository.spec.js

```diff
@@ -684,6 +684,36 @@ describe('SVDocumentMongoDbRepository', function main() {
       expect(collectionsAfter).to.have.lengthOf(1);
       expect(collectionsAfter[0].collectionName).to.equal(svDocumentRepository.getCollectionName());
     });
+
```

Comment:
We need to test `createIndices` as well. In `createCollection` you can check that you just call `createIndices()`

Reply:
We check this in the next test

Reply:
I don't see: `describe('#createIndices')` in this suite.